src/utils/time.ts

```diff
@@ -18,6 +18,34 @@ export function getCurrentTimeInHighPrecision(): string {
   return Temporal.Now.instant().toString({ smallestUnit: 'microseconds' });
 }
 
+/**
+ * Creates a UTC ISO-8601 timestamp in microsecond precision accepted by DWN.
+ * @param year
+ * @param month
+ * @param day
+ * @param hour
+ * @param minute
+ * @param second
+ * @param millisecond
+ * @param microsecond
+ * @returns string
+ */
+export function createTimestamp(
+  year: number, month: number, day: number, hour: number, minute: number, second: number, millisecond: number, microsecond: number
```

Comment:
All parameters should be optional for this function just like the `from` method in the `Temporal` lib used in this implementaiton.